android/app/src/main/AndroidManifest.xml

```diff
@@ -31,12 +31,6 @@
         android:launchMode="singleTask"
         android:exported="true"
         android:windowSoftInputMode="adjustResize">
-      </activity>
-
-      <activity
```

Comment:
Remove the need for a custom Activity since `react-native-bootsplash` v4 uses Android `SplashScreen` API.